### ISSUE TITLE
When Playing BRSTM after full loop audio studders and RAM usage soars. (OpenAL)

### ISSUE THREAD

**Issue by The-Silver-Kid** [libertyernie/brawltools#178]

**Comment by libertyernie** [comment]
It only happens after the audio loops, right? 

Are you on Windows? Also check the Help > About screen and let me know if it says what audio backend it's using.

**Comment by The-Silver-Kid** [comment]
Yes it only happens after one full loop and continues playing after that.

**Comment by libertyernie** [comment]
BrawlBox usually tries DirectSound first, and only uses OpenAL if that fails. I wrote the OpenAL wrapper code but it's kind of buggy.

Are you building BrawlBox yourself from Visual Studio? If you are, try changing the target platform to x86.

**Comment by The-Silver-Kid** [comment]
Setting it to build for x86 does fix the problem where it doesn't use DirectSound.
OpenAL still has issues.

**Comment by The-Silver-Kid** [comment]
Setting it to build for x86 does fix the problem where it doesn't use DirectSound.
OpenAL still has issues.

**Comment by The-Silver-Kid** [comment]
Changing target platform to x86 fixes not detecting (or using) DirectSound.  But does not fix the OpenAL bug.

**Comment by libertyernie** [comment]
I'll keep that in mind, thanks.

**Comment by The-Silver-Kid** [comment]
Has this been fixed in .78 Hotfix 1?
